dark-node/dark_node_test.go

```diff
@@ -25,7 +25,7 @@ import (
 
 const (
 	NumberOfBootstrapNodes = 5
-	NumberOfOrders         = 100
+	NumberOfOrders         = 10
```

Comment:
I reduced the number of orders to 10 (from 100) and the number of non-bootstrap nodes to 5 (from 15) to make the tests more lightweight (before they would take a few minutes and occasionally timeout)